exporter/alibabacloudlogserviceexporter/metricsdata_to_logservice.go

```diff
@@ -51,7 +51,14 @@ func (kv *KeyValues) Swap(i, j int) {
 	kv.keyValues[i], kv.keyValues[j] = kv.keyValues[j], kv.keyValues[i]
 }
 func (kv *KeyValues) Less(i, j int) bool { return kv.keyValues[i].Key < kv.keyValues[j].Key }
-func (kv *KeyValues) Sort()              { sort.Sort(kv) }
+
+func (kv *KeyValues) Sort() {
+	// sanitize label keys before sort
```

Comment:
Shouldn't sanitization happen on insertion, not sorting? It seems mysterious for a sort method to modify the data itself.

Reply:
done, sanitize keys when insert